### DISCUSSION TITLE
Is there any plans to open source the Learn section of nextjs.org?

### DISCUSSION THREAD

**Discussion by seriouscatsoserious** [vercel/next.js#48172] [Help]
### Summary

I'm a brand new user and learning Next via the docs, I've noticed several spelling errors and have a few issues open about them. I'd love to just jump in and put up a PR for them myself but the Learn section isn't open. Does anyone know if there's plans for this or why this is?

**Comment by ghost** [comment]
I would love to have access to Learn in github as well.

**Comment by icyJoseph** [comment]
Far from an answer but you can create issues here,  https://github.com/vercel/next-learn hopefully someone picks it up and fixes the things you found.
